### ISSUE TITLE
IFC Geometry

### ISSUE THREAD

**Issue by MarkAckerley** [erfajo/OrchidForDynamo#20]
Hi Erik,

I would like to be able to access the geometry of an IFC file without loading it into Revit.

I imagine a node with an input of file path and outputs of Revit elements &/or solids/meshes

The desired workflow is to 3D print IFC files without requiring Revit.

I see potential for custom IFC workflows which improve IFC operability.

I don't know if this is remotely possible, but if I don't ask I'll never get :) 

Thanks a lot,

Mark

**Comment by erfajo** [comment]
I agree it would be good that there is some IFC nodes. However, this is a new ballgame I have to consider how to do before I can say something meaningful about it :-)
